.changeset/smart-sloths-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
+---
+
+The auth0 integration is updated to use the passport-auth0 library. The configuration under auth.providers.auth0.* now supports an optional audience parameter; providing that allows you to connect to the correct API to get permissions, access tokens, and full profile information.
```

Comment:
note that you can put backticks around identifiers etc, which makes for a better read and also circumvents vale spell checks. So ``` `passport-auth0` ```, ``` `auth.providers.auth0.*` ```, ``` `audience` ```

Reply:
done there is always something, should there be consideration for something like `husky` and `lint-staged` to enforce the linting of items before they get pushed?